Sources/DiskCache/Cache.swift

```diff
@@ -7,16 +7,49 @@
 
 import Foundation
 
+/// A class of types providing interfaces for caching and retrieving data to/from disk.
 public protocol Cache {
+
+    /// Synchronously writes `data` to disk.
+    /// - Parameters:
+    ///   - data: The data to write to disk.
+    ///   - key: A unique key used to identify `data`.
     func syncCache(_ data: Data, key: String) throws
+
+    /// Synchronously get data from the cache. If data does not exist for `key`, an error will be thrown.
```

Comment:
Copied from `DiskCache`, though I did omit that the thrown error would have the `NSFileReadNoSuchFileError`. I don't feel strongly about it, but it seems to me that that is outside the ability of a protocol to specify until / if we get typed throws.